contracts/mainnet/connectors/euler/euler-rewards/main.sol

```diff
@@ -0,0 +1,46 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+import "./helpers.sol";
+import "./events.sol";
+
+/**
+ * @title Euler Rewards.
+ * @dev Claim Euler rewards.
+ */
+
+contract EulerIncentives is Helpers, Events {
+
+    /**
+     * @dev Claim Pending Rewards.
+     * @notice Claim Pending Rewards from Euler incentives contract.
+     * @param user Address that should receive tokens.
+     * @param token Address of token being claimed (ie EUL)
+     * @param amt The amount of reward to claim.
+     * @param proof Merkle proof that validates this claim.
+     * @param getId ID to retrieve amt.
+     * @param setId ID stores the amount of rewards claimed.
+    */
+    function claim(
+        address user,
+        address token,
+        uint256 amt,
+        bytes32[] memory proof,
+        uint256 getId,
+        uint256 setId
+    ) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        uint _amt = getUint(getId, amt);
```

Comment:
@shriyatyagii We can claim partial amount also?

Reply:
No, as it will result to invalid hash.